o2dpg-sim-tests.sh

```diff
@@ -31,9 +31,15 @@ O2DPG_TEST_ANALYSISQC_EXITCODE=0
 { O2DPG_TEST_REPO_DIR=${SOURCEDIR}/../../../O2DPG/${O2DPG_VERSION}/0 "${O2DPG_ROOT}/test/run_analysisqc_tests.sh" >> ${LOGFILE} 2>&1 ; O2DPG_TEST_ANALYSISQC_EXITCODE=$?; } || true  # don't quit immediately on error
 # only log/test files created, nothing to delete here
 
+O2DPG_TEST_RELVAL_EXITCODE=0
+{ O2DPG_TEST_REPO_DIR=${SOURCEDIR}/../../../O2DPG/${O2DPG_VERSION}/0 "${O2DPG_ROOT}/test/run_relval_tests.sh" >> ${LOGFILE} 2>&1 ; O2DPG_TEST_RELVAL_EXITCODE=$?; } || true  # don't quit immediately on error
+# keep only logs, remove everything else for now
+# if we needed/possible in the future, we might keep some of the other files
+find . -type f ! -name '*.log' -delete
```

Comment:
If you are running in BUILD, I think there is no need to remove the logs, given they do not get packaged in any case. Just make sure you delete them at the beginning, so that we start from a clean slate.